code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -202,10 +202,20 @@
 		if(locate(/obj/item/pillow) in owner.loc)
 			healing += 0.1
 
-		if(healing > 0 && health_ratio > 0.8)
-			owner.adjustBruteLoss(-1 * healing, required_bodytype = BODYTYPE_ORGANIC)
-			owner.adjustFireLoss(-1 * healing, required_bodytype = BODYTYPE_ORGANIC)
-			owner.adjustToxLoss(-1 * healing * 0.5, TRUE, TRUE, required_biotype = MOB_ORGANIC)
+		if(healing > 0)
+			// gives a boost to internal organs healing
+			for(var/obj/item/bodypart/target_bodypart in owner.bodyparts)
```

Comment:
Why iterate over all bodyparts anyways, why not just iterate over `owner.organs`

Reply:
Good question

Reply:
Applied with e5c590f6d66c1f2d0f7d46a11c3b2f9b8c6bc1b2